index.js

```diff
@@ -72,12 +88,20 @@ function isEmail(email) {
     return re.test(String(email).toLowerCase())
 }
 
-function getEmailFromIdentifyEvent(event) {
-    return isEmail(event.distinct_id)
-        ? event.distinct_id
-        : !!event['$set'] && Object.keys(event['$set']).includes('email')
-        ? event['$set']['email']
-        : ''
+function getEmailFromEvent(event) {
```

Comment:
Should there be a check for `$email` as well? We had a discussion about whether that property should/shouldn't be in our taxonomy going forward, but I don't remember what the conclusion was.

Reply:
I took a look at the JSON for some identify and `user signed up` events (what we're going to use to trigger this) and didn't see a `$email` property - only `email` so opted to leave it out sans example.